native-sql-engine/core/src/main/scala/com/intel/oap/execution/ColumnarShuffledHashJoinExec.scala

```diff
@@ -55,6 +55,57 @@ import org.apache.spark.sql.catalyst.optimizer.{BuildLeft, BuildRight, BuildSide
 import org.apache.spark.sql.execution.joins.{HashJoin,ShuffledJoin,BaseJoinExec}
 import org.apache.spark.sql.execution.joins.HashedRelationInfo
 
+trait ColumnarShuffledJoin extends BaseJoinExec {
```

Comment:
The code for ColumnarShuffledJoin is as same as the code for ShuffledJoin of spark 3.2, except that ColumnarShuffledJoin directly extends BaseJoinExec, but ShuffledJoin indirectly extends BaseJoinExec.